.github/workflows/merge_to_master.yml

```diff
@@ -107,3 +109,44 @@ jobs:
           file: ./Dockerfile
           push: true
           tags: ${{ secrets.QUAY_SERVER }}/${{ secrets.QUAY_NAMESPACE }}/robottelo:${{ steps.image_tag.outputs.IMAGE_TAG }}
+
+
+  upgrades_robottelo_container:
+    needs: codechecks
+    name: Update Upgrades Robottelo container image on Quay.
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get image tag
+        id: image_tag
+        run: |
+          echo -n ::set-output name=IMAGE_TAG::
+          TAG="${GITHUB_REF##*/}"
+          if [ "${TAG}" == "master" ]; then
+              TAG="latest"
+          fi
+          echo "${TAG}"
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Quay Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ secrets.QUAY_SERVER }}
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: Build and push image to Quay
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
```

Comment:
This container is already built in this workflow.

The secrets for access to quay are not available in workflows executed against PR's from forks.

Reply:
I am closing this PR as its not required ! I need your ACK on Upgrade build container!